nengo_bones/templates/pyproject.toml.template

```diff
@@ -0,0 +1,16 @@
+# Automatically generated by nengo-bones, do not edit this file directly
+
+[build-system]
+requires = ["setuptools", "wheel"]
```

Comment:
What is this part for?

Reply:
`pyproject.toml` is a general file for doing Python project configuration; it can completely replace `setup.cfg` and `setup.py` using tools like [Flit](https://flit.readthedocs.io/en/latest/) or [Poetry](https://poetry.eustace.io/). This little piece of metadata tells `pip` that, despite having a `pyproject.toml` file, we are still using the normal `setuptools` and `wheel` tools to distribute our packages. It's part of the [PEP-518 spec](https://www.python.org/dev/peps/pep-0518/#build-system-table).

It is optional, though, we can remove it if we don't want to specify it. It's then undefined what tools reading `pyproject.toml` do if they want to infer the build system. There is, unfortunately, no other way to configure Black.